flang/lib/Lower/FIRBuilder.cpp

```diff
@@ -38,6 +38,14 @@ mlir::Type Fortran::lower::FirOpBuilder::getRefType(mlir::Type eleTy) {
   return fir::ReferenceType::get(eleTy);
 }
 
+mlir::Value
+Fortran::lower::FirOpBuilder::createNullConstant(mlir::Location loc) {
+  auto indexType{this->getIndexType()};
+  auto zero{this->createIntegerConstant(loc, indexType, 0)};
+  auto noneRefType{this->getRefType(this->getNoneType())};
```

Comment:
Please follow the MLIR coding conventions as in the rest of this file. `auto x = ...;`